### PR TITLE
Release 0.6.0; Changes to packaging only;

### PR DESCRIPTION
**Pull request by odscjames**
- Removed all requirements files and moved to setup.py
- Moved data underneath main folder where it can be included in package
- Fixed repo URL in setup.py

Also

- Added new Python versions to Travis

### CODE REVIEW

**Comment by odscjames** [comment]
So this is ok, but I had to remove 2 tests to make this work. My first suspect is that is to do with the upgrade from jsonschema from 2.6 to 3.2

File: setup.py [hunk 1]
```diff
@@ -2,33 +2,26 @@
...
-        'bleach',
-        'requests',
-        'json-merge-patch',
-        'cached-property',
         'python-dateutil',
-        # TODO Should also have flatten-tool >= v0.5.0 - that is currently in requirements instead.
-        # TODO Should also have lib-cove  >= v0.3.1 - that is currently in requirements instead.
+        'Django>1.11.23,<1.12',
+        'flattentool>=0.5.0',
+        'libcove>=0.6.0'
```
Comment (stevenday) [inline setup.py]: I went to v0.13.0 here, because before that libcove didn't specify all its dependencies, that might be a clue to the specs?

**Comment by Bjwebb** [comment]
`rfc3987` and `strict-rfc3339` are no longer installed, which are required to validate URIs and date-times. These should be dependencies of lib-cove. I'll create a pull request there.

**Comment by Bjwebb** [comment]
I've made a lib-cove PR https://github.com/OpenDataServices/lib-cove/pull/48

**Comment by Bjwebb** [comment]
This now requires the latest lib-cove, so that we get those optional dependencies of jsonschema.

I think this is now ready to merge. @stevenday could you check?

**Comment by stevenday** [comment]
Thanks @Bjwebb, I've tested this locally, in the register and in our CI and it all looks good 👍 

With this, I think we can also update the FAQ in the README regarding local installation, it should now be as simple as:

```
python3 -m venv .ve
source .ve/bin/activate
pip install -e .
```

We can also close #15 and #19 I think?

**Comment by Bjwebb** [comment]
I've updated the README. I've removed the FAQ, and moved the installation steps further up, so hopefully people shouldn't run into that same issue any more.

**Pull request merged**